src/components/Profile/InstructorProfile.tsx

```diff
@@ -36,52 +38,11 @@ export const InstructorProfile: React.FC = () => {
             .catch(err => {
                 console.log(err);
             });
-    }, [authourId]);
-
-    const courseData = [
-        {
-            img:
-                'https://miro.medium.com/max/1400/1*uvd7Z4npUG8qulaQLjHcZw.jpeg',
-            title: 'GraphQL Advanced Course',
-            rate: 4.5,
-            likes: 1000,
-            enrolled: 531,
-        },
-        {
-            img:
-                'https://miro.medium.com/max/2880/1*xcDT-neKHP7E3quS9n30gw.png',
-            title: 'React Hooks Course',
-            rate: 5,
-            likes: 1500,
-            enrolled: 321,
-        },
-        {
-            img:
-                'https://railsware.com/blog/wp-content/uploads/2018/09/2400%D1%851260-rw-blog-node-js.png',
-            title: 'Nodejs Advanced Course',
-            rate: 4,
-            likes: 784,
-            enrolled: 212,
-        },
-        {
-            img: 'https://i.ytimg.com/vi/OdU9H-_d14Y/maxresdefault.jpg',
-            title: 'React Native With Typescript',
-            rate: 4,
-            likes: 1000,
-            enrolled: 651,
-        },
-        {
-            img:
-                'https://dist.neo4j.com/wp-content/uploads/20170524234854/graph-ql-graph-database-neo4j-integration.png',
-            title: 'GraphQL with Neo4j  Course',
-            rate: 5,
-            likes: 1000,
-            enrolled: 531,
-        },
-    ];
+    }, [authorID]);
+
     return (
         <ScrollView style={[styles.wrapper, { backgroundColor: colors.background }]}>
-            <Async promise={getCheckedUserInfo(authourId)}>
+            <Async promise={getCheckedUserInfo(authorID)}>
                 {
                     ({ data, error, isLoading }) => {
```

Comment:
you can write 
```
(getUserInfo) = {
     if(getUserInfo.loading) return <Text>Loading...</Text>
}
```

that way will be better than this form 
```
 ({ data, error, isLoading }) => {
if (isLoading) return <View><Text>loading</Text></View>}
```
cuz if we use <Query/> inside <Async/> data will be the same